OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -146,6 +146,10 @@ public void Draw()
 				if (a.Actor.IsInWorld && !a.Actor.Disposed)
 					a.Trait.RenderAfterWorld(this, a.Actor);
 
+            //added for Weathereffects
```

Comment:
Proper comments start with a space and a capital letter: `// Added for WeatherEffects.`